Formula/clblast.rb

```diff
@@ -0,0 +1,19 @@
+class Clblast < Formula
+  desc "Tuned OpenCL BLAS library"
+  homepage "https://github.com/CNugteren/CLBlast"
+  url "https://github.com/CNugteren/CLBlast/archive/1.3.0.tar.gz"
+  sha256 "fd5e06c261551366b46ccbc7841012cc69b58c74ed641db4f7a157c055d24223"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    pkgshare.install "samples/sgemm.c" # for a simple library linking test
```

Comment:
Let's install the whole samples directory

Reply:
OK, I did that. And I adjusted the test to `pkgshare/"samples\/sgemm.c"`, I guess I have to escape the slash?

Reply:
Not unless it's a regex, which it isn't.

Reply:
OK, fixed it.